cli/internal/oauth/util/browser.go

```diff
@@ -0,0 +1,37 @@
+package util
+
+import (
+	"errors"
+	"os/exec"
+	"runtime"
+	"strings"
+)
+
+// OpenBrowser opens the specified URL in a browser based on OS.
```

Comment:
Just a really short blurb; wondering if we looked at https://github.com/pkg/browser - I know that one is used already by various dependencies we use for authentication purposes, so may be battle tested (I'm not sure how much different it is from the code here though)